tensorrt_llm/bench/benchmark/throughput.py

```diff
@@ -253,6 +254,12 @@ def throughput_command(
     """Run a throughput test on a TRT-LLM engine."""
 
     logger.info("Preparing to run throughput benchmark...")
+
```

Comment:
Is there a reason why these aren't enumerations at this point? We seem to keep changing the naming. An enumeration would help with keeping a consistent interface.

Reply:
Can you elaborate on this?

Reply:
Looking at this, this is just to stylize the text. I was looking at this on my phone earlier, so I thought it was potentially being used elsewhere. This is fine I think. The thing I was thinking of was a `MultiValue` string enum:

```python
from aenum import MultiValueEnum

class BackendEnum(MultiValueEnum):
    PYTORCH = "PyTorch", "pytorch", "pyt"
    TENSORRT = "TensorRT", "tensorrt", "trt"
    AUTODEPLOY = "AutoDeploy", "_autodeploy", "ad", "autodeploy"

    @property
    def display_name(self) -> str:
        return self.value
```